src/i_input.c

```diff
@@ -445,6 +449,32 @@ static int AccelerateMouseY(int val)
     }
 }
 
+// [crispy] Distribute the mouse movement between the current tic and the next
+// based on how far we are into the current tic. Compensates for mouse sampling
+// jitter.
+static void SmoothMouse(int* x, int* y)
+{
+    static int x_remainder_old = 0;
+    static int y_remainder_old = 0;
+    int x_remainder, y_remainder;
+    fixed_t correction_factor;
+    fixed_t fractic;
+
+    *x += x_remainder_old;
+    *y += y_remainder_old;
+
+    fractic = (int64_t)I_GetTimeMS() * TICRATE % 1000 * FRACUNIT / 1000;
```

Comment:
Could you please factor this out into a helper function as we have done here?

https://github.com/fabiangreffrath/woof/blob/b28aee2679692111c14e9c65f81a8a1054fb2c17/Source/i_system.c#L118